ci-operator/config/heketi/heketi/heketi-heketi-master.yaml

```diff
@@ -0,0 +1,29 @@
+
+canonical_go_repository: github.com/heketi/heketi
+
+base_images:
+  os:
+    cluster: https://api.ci.openshift.org
+    name: centos
+    namespace: openshift
+    tag: '7'
+
+build_root:
+  project_image:
+    dockerfile_path: extras/openshift-ci/build-image/Dockerfile
+
+binary_build_commands: make
```

Comment:
You're not using the output of this for anything, so it will not be run -- is that intentional?

Reply:
Probably not, if we're only using the openshift-ci initially for testing, is this something that I should remove?

Reply:
No, we can keep it -- just as long as you know it won't get run.